test/CMakeLists.txt

```diff
@@ -176,5 +176,7 @@ endif()
 
 add_custom_command(TARGET runtests
                    COMMENT "Run tests"
-                   POST_BUILD COMMAND ${COVERAGE_PREFIX} ${CMAKE_CTEST_COMMAND} --output-on-failure -C $<CONFIG>
+                   POST_BUILD
+                   COMMAND "$<TARGET_FILE:test-sfml-window>" "--test-case=[Window] sf::Context" "--subcase=Version String" -m -ne -ni -nv -nc
+                   COMMAND ${COVERAGE_PREFIX} ${CMAKE_CTEST_COMMAND} --output-on-failure -C $<CONFIG>
                    VERBATIM)
```

Comment:
How is this going to complicate #2430? Can I still run all the tests by running `ctest` or does this mean I need to invoke special alternative command now? I hope running our tests locally is never more complicated than simply invoking `ctest`.

Reply:
Nothing changes for people running ctest themselves. The first `COMMAND` line was just added for OpenGL implementation verification purposes when we build in CI.